apps/developer-hub/src/components/NotFound/index.module.scss

```diff
@@ -0,0 +1,57 @@
+@use "@pythnetwork/component-library/theme";
```

Comment:
There's also a [NotFoundPage component in the component library](https://github.com/pyth-network/pyth-crosschain/tree/main/packages/component-library/src/NotFoundPage) now which I suggest you move to -- sorry I didn't have these extracted before you started working on this!